src/azul/logging.py

```diff
@@ -8,6 +10,14 @@
 
 def configure_app_logging(app: AzulChaliceApp, *loggers):
     _configure_log_levels(app.log, *loggers)
+    # enable logger for elasticsearch
+    es_trace_logger = logging.getLogger('elasticsearch')  # get built in logger from ES client
+    es_trace_logger.setLevel(logging.DEBUG)  # set level to Debug
+    es_trace_logger.handlers.clear()  # clear any null handlers
+    es_trace_formatter = logging.Formatter('%(asctime)s %(levelname)-7s %(threadName)s: ES request: %(message)s')
+    handler = logging.StreamHandler(stream=sys.stderr)  # instantiate stream handler
+    handler.setFormatter(es_trace_formatter)  # set formatter
+    es_trace_logger.addHandler(handler)  # set handler
```

Comment:
Consider putting all of the Elasticsearch logging code in its own function within this module. Possibly titled `configure_es_logging()`.

Reply:
All of these comments tell me *what* is happening, which I can already deduce from reading the code. They should be removed.

Comments are especially helpful in explaining *why* certain things are done, especially when it's not obvious. For example, why do we need to set a new handler here, instead of just letting the logs propagate to the root handler (and do we actually need to)? I suspect that removing the null handler would be sufficient, but I would have to test this.

Reply:
You configure the logging only for "apps", but I don't think that's the right place. "app" logging applies to any code that's run inside of a lambda function, but we still make calls with the Elasticsearch client in side of scripts and other places. Moving this code (or the call site for `configure_es_logging()` within `_configure_log_levels()` would solve this.